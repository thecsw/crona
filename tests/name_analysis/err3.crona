main : void() {
    a : int array[1];
    a : void;
    a[10] = 9;
}