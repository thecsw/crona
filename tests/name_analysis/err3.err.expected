FATAL [3,5]: Invalid type in declaration
FATAL [3,5]: Multiply declared identifier
Name Analysis Failed
